### ISSUE TITLE
Option last_segment_only results in dublicate path part or slashes in slug

### ISSUE THREAD

**Issue by datango** [wazum/sluggi#28]
This is a great option, but it has some problems:

1)  Dublicate page slug in subpath:
    Given: [https://servername.de/demo/test]/page1
	Changing page1 to page1new
	Result: [https://servername.de/demo/demo/test]/page1new
	Changing again to page1new2
	Result: [https://servername.de/demo/demo/test]/page1new2
	
2) Problems when using slug as admin

	Given: [https://servername.de]/demo/test/page1
	Changing page1 to page1new
	Result: [https://servername.de]/demo/test/demo-test-page1new and error message 
	"You are not allowed to insert slashes into the URL!" (message.slashesNotAllowed)

**Comment by wazum** [comment]
@datango  what version did you use?

**Comment by datango** [comment]
@wazum 1.9.10

**Comment by wazum** [comment]
@datango  so please use the latest version

**Comment by datango** [comment]
@wazum: Thanx for the hint. I only searched the TER. 

My second problem "using slug as admin" is gone with update to version 1.9.11. But the dublicates as user are still processed.
> Result: [https://servername.de/demo/demo/test]/page1new

I have a fix that works for me: 

[sluggifixdublicates.txt](https://github.com/wazum/sluggi/files/4613572/sluggifixdublicates.txt)

I removed the $inaccessibleSlugSegments variable in the DatamapHook for the $incomingFieldArray['slug'].

(i use TYPO3 version 9)
